janusgraph-core/src/main/java/org/janusgraph/graphdb/database/IndexSerializer.java

```diff
@@ -355,8 +356,10 @@ public void reindexElement(JanusGraphElement element, MixedIndexType index, Map<
                 element.values(key.name()).forEachRemaining(value->entries.add(new IndexEntry(key2Field(field), value)));
             }
         }
-        final Map<String, List<IndexEntry>> documents = documentsPerStore.computeIfAbsent(index.getStoreName(), k -> Maps.newHashMap());
```

Comment:
I'm not really familiar with this part of the code base, but could you quickly explain why this isn't necessary any more?

Reply:
Hi @FlorianHockmann,
This code is removed for 2 reasons.
1.  `getDocuments(documentsPerStore, index)` does exact same thing, and hence it was duplicate execution of `computeIfAbsent`
2. `documents` variable was not being used.

Reply:
Thanks for the explanation! That makes sense of course.